pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -1,9 +1,9 @@
 { stdenv, lib, fetchFromGitHub, fetchpatch, buildPythonPackage, python,
-  cudaSupport ? false, cudatoolkit, cudnn, nccl, magma,
+  cudaSupport ? false, cudaPackages, magma,
   mklDnnSupport ? true, useSystemNccl ? true,
   MPISupport ? false, mpi,
   buildDocs ? false,
-  cudaArchList ? null,
+  cudaArchList ? [],
```

Comment:
Ah, here it is: `final_cudaArchList` used to be assigned some defaults, conditioned on `cudaArchList` being `null`

Reply:
Right, but being it a list it would make sense to have the default here be a list. We can fix the final list.

Reply:
Appears to be a `Maybe List`